backend/src/helpers/VeridaHelper.ts

```diff
@@ -183,7 +183,7 @@ export default class VeridaHelper {
 
       return json;
     } catch (error) {
-      console.log({ error });
+      throw new Error(error);
```

Comment:
If you just throwing the same error further you could simply delete the try..catch

Reply:
resolved